src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  currentPage: number;
+  items: string[];
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  items,
+  perPage,
+  onPageChange,
+}) => {
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < Math.ceil(items.length / perPage)) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+            disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
```

Comment:
Using `href` attributes with values like `#prev` can cause the page to jump to the top when clicked. Consider using `href="#"` and calling `event.preventDefault()` in the `onClick` handler to prevent this behavior.